extensions/mssql/src/features.ts

```diff
@@ -27,6 +29,52 @@ export class TelemetryFeature implements StaticFeature {
 	}
 }
 
+export class AccountFeature implements StaticFeature {
+
+	constructor(private _client: SqlOpsDataClient) { }
+
+	fillClientCapabilities(capabilities: ClientCapabilities): void {
+		// this isn't explicitly necessary
+	}
+
+	initialize(): void {
```

Comment:
@Charles-Gagnon do we want a telemetry notification when we get an AE request? Would it help to see adoption?

Reply:
Yeah good call, it looks like this is using the older telemetry library though so once this is merged in I'll go through and update that and add the events in.